src/org/zaproxy/zap/extension/AllInOneNotes/NotesTableModel.java

```diff
@@ -0,0 +1,49 @@
+package org.zaproxy.zap.extension.AllInOneNotes;
+
+import javax.swing.table.AbstractTableModel;
+import java.util.List;
+import java.util.Vector;
+
+public class NotesTableModel extends AbstractTableModel {
+
+    private final Vector<String> columnNames;
+    private static final int COLUMN_COUNT = 2;
+    private String[][] rowData;
+
+    public NotesTableModel(List<String[]> data){
+        super();
+
+        columnNames = new Vector<>(COLUMN_COUNT);
+        columnNames.add("Request ID");
+        columnNames.add("Note Content");
```

Comment:
These should be i18n from the Messages.properties file (see the button comment earlier).

Reply:
agreed! changed